projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectLinkingService.java

```diff
@@ -0,0 +1,73 @@
+package life.qbic.projectmanagement.application;
+
+import static life.qbic.logging.service.LoggerFactory.logger;
+
+import java.util.Optional;
+import java.util.UUID;
+import life.qbic.logging.api.Logger;
+import life.qbic.projectmanagement.application.finances.offer.OfferSearchService;
+import life.qbic.projectmanagement.domain.finances.offer.Offer;
+import life.qbic.projectmanagement.domain.project.OfferIdentifier;
+import life.qbic.projectmanagement.domain.project.Project;
+import life.qbic.projectmanagement.domain.project.ProjectId;
+import life.qbic.projectmanagement.domain.project.repository.ProjectRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * Allows for modification of existing projects
+ *
+ * @since 1.0.0
+ */
+@Service
+public class ProjectLinkingService {
+
+  Logger log = logger(ProjectLinkingService.class);
+  private final ProjectRepository projectRepository;
+  private final OfferSearchService offerSearchService;
+
+  public ProjectLinkingService(
+      @Autowired ProjectRepository projectRepository,
+      @Autowired OfferSearchService offerSearchService) {
+    this.projectRepository = projectRepository;
+    this.offerSearchService = offerSearchService;
+  }
+
+  public void linkOfferToProject(String offerIdentifier, String projectIdentifier) {
+    log.info("linking offer " + offerIdentifier + " to project " + projectIdentifier);
+    Offer offer = loadOfferOrThrow(offerIdentifier);
+    Project project = loadProjectOrThrow(projectIdentifier);
+    OfferIdentifier offerIdentifier2 = OfferIdentifier.of(offer.offerId().id());
+    project.linkOffer(offerIdentifier2);
```

Comment:
NIT: We already have the ID and have proven that the ID exists so there is no need for a new offeridentifier construction just to get the same ID 
```suggestion
    project.linkOffer(OfferIdentifier.of(offerIdentifier));
```